src/scss/includes/masqAvatar.scss

```diff
@@ -0,0 +1,19 @@
+.masqAvatar {
+  overflow: hidden;
+
+  &-letter {
+    color: white;
+    width: 100%;
+    height: 100%;
+    font-weight: bold;
+    font-size: 25px;
+    line-height: 1;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &-image {
+    width: 100%;
+  }
+}
```

Comment:
why a new file?

Reply:
Because it is nice for each single re-usable React component to have a single corresponding Scss file, with coherent naming. That way, it's easier to scope and manage.

It would also allow to import only the SCSS file needed by a component, directly from the JSX file, if we decide to change our CSS building process (for now we build a single big file with Sass, with many unused rules, but there are other ways).